src/MoBi.Assets/AppConstants.cs

```diff
@@ -860,6 +860,7 @@ public static class Dialog
          public static readonly string ApplyDefaultNaming = "Apply default renaming";
          public static readonly string RemoveSelectedResultsFromSimulations = "Do you really want to remove the selected results from simulations";
          public static readonly string RemoveSelectedResultsFromProject = "Do you really want to remove the selected result(s) from the project";
+         public static readonly Size SELECT_SINGLE_SIZE = new Size(475, 160);
```

Comment:
all the sizes are defined in another file no ? UICOnstants or sthg? or am I wrong

Reply:
Yeah, but in this case we want to set the size of the modal from the presentation layer. UIConstants is in MoBi.UI. I agree this is a little out of place, but I don't think I can put it in UIConstants.

Maybe a separate static class in this file because it's a bit out of place with these strings.

Reply:
why do we set the size from the presentation layer?

Reply:
This is a modal that encapsulates another view. So the pattern is to resolve this modal in a presenter, call encapsulate with another presenters view. The modal has no idea what's there.